c2corg_ui/templates/image/edit.html

```diff
@@ -0,0 +1,316 @@
+              <div class="help-block" ng-messages="editForm.focal_length.$error">
+                <p ng-message="number" class="label label-danger" style="{{ (editForm.focal_length.$invalid && editForm.focal_length.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ## EXPOSURE TIME
+            <div class="form-group half" ng-class="{ 'has-error': editForm.exposure_time.$invalid, 'has-success': editForm.exposure_time.$valid && image.exposure_time }">
+              <label translate>exposure_time</label>
+              <div class="input-group">
+                <span class="input-group-addon">1/</span>
+                <input type="number" min="1" name="exposure_time" ng-model="image.exposure_time" class="form-control">
+                <span class="input-group-addon">s</span>
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.exposure_time.$valid && image.exposure_time"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.exposure_time.$invalid && editForm.exposure_time.$touched"></span>
+              <div class="help-block" ng-messages="editForm.exposure_time.$error">
+                <p ng-message="number" class="label label-danger" style="{{ (editForm.exposure_time.$invalid && editForm.exposure_time.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ## ISO
+            <div class="form-group half" ng-class="{ 'has-error': editForm.iso_speed.$invalid, 'has-success': editForm.iso_speed.$valid && image.iso_speed }">
+              <label translate>iso_speed</label>
+              <div class="input-group">
+                <span class="input-group-addon">ISO</span>
+                <input type="number" min="1" name="iso_speed" ng-model="image.iso_speed" class="form-control">
+              </div>
+              <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="editForm.iso_speed.$valid && image.iso_speed"></span>
+              <span class="glyphicon glyphicon-warning-sign form-control-feedback" ng-show="editForm.iso_speed.$invalid && editForm.iso_speed.$touched"></span>
+              <div class="help-block" ng-messages="editForm.iso_speed.$error">
+                <p ng-message="number" class="label label-danger" style="{{ (editForm.iso_speed.$invalid && editForm.iso_speed.$touched) ? 'display:inline' : ''}}" translate>This field must be a number.</p>
+              </div>
+            </div>
+
+            ## WIDTH
+            <div class="form-group half" ng-if="image.width">
+              <label translate>width</label>
+              <div class="input-group">
+                <input type="number" disabled min="0" minlength="1" name="width" ng-model="image.width" class="form-control" />
```

Comment:
they are shown only if the property already exists within the object.  If yes, it will appear in a `disabled` input, so the user cannot edit it in any case.